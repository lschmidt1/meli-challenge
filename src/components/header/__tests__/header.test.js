import { shallow, render } from "enzyme";
import Header from "../header";

describe("<Header />", () => {
  describe("should render correctly", () => {
    test("by matching snapshot", () => {
      const { container } = render(<Header />);
      expect(container).toMatchSnapshot();
    });
  });
  test("button click should execute onSubmit function", () => {
    const clickFn = jest.fn();
    const component = shallow(<button onClick={clickFn}></button>);
    expect(component.find("button").exists()).toBe(true);
    component.find("button").simulate("click");
    expect(clickFn).toHaveBeenCalled();
  });
});

/*   it('should render banner text correctly with given strings', () => {
    const strings = ['one', 'two'];
    const component = shallow(<MyComponent list={strings} />);
    expect(component).toMatchSnapshot();
  }); */
