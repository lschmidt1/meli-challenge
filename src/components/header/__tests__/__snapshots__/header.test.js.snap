// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render correctly by matching snapshot 1`] = `undefined`;
